extra/exe-builder/Program.cs

```diff
@@ -204,8 +225,13 @@ public class UptimeKumaApplicationContext : ApplicationContext
 
         }
 
-        void VisitGitHub(object sender, EventArgs e)
-        {
+        void VisitGitHub(object sender, EventArgs e) {
+            if (intranetOnly)
+            {
+                MessageBox.Show("You have parsed in --intranet so we will not try to access the internet.");
```

Comment:
`The --intranet argument was provided, so we will not try to access the internet.` should already be in the log => I don't see why logging this again has additional value

```suggestion
```